### DISCUSSION TITLE
PR to Mill contrib docker plugin or third party plugin

### DISCUSSION THREAD

**Discussion by GeorgOfenbeck** [com-lihaoyi/mill#3139] [Development]
I love mill and I use it a lot atm to create images for ARM while being on my amd64.
To do so I modified the docker contrib module a tiny bit
[https://github.com/com-lihaoyi/mill/compare/main...GeorgOfenbeck:mill:feature/add-docker-buildx](https://github.com/com-lihaoyi/mill/compare/main...GeorgOfenbeck:mill:feature/add-docker-buildx)

Would that tiny change be enough for a pull request?

I can of course also make it a standalone third party contribution as e.g done for the
[native image](https://github.com/carlosedp/mill-docker-nativeimage)
but seems a bit overkill for the tiny change.

What would be preferred?

**Comment by lefou** [comment]
Great that you enjoy the Mill experience and welcome to the community! 

I think your extension would make a nice pull request. It would be good, to also update the documentation (`contrib/doccker/readme.adoc`, which is renderer to [this page](https://mill-build.com/mill/contrib/docker.html)) accordingly.

**Comment by GeorgOfenbeck** [comment]
Thanks for the feedback - PR https://github.com/com-lihaoyi/mill/pull/3143 created
